### PR TITLE
BritishBroadcastingCorporation.Iplayer version 2.13.2

### PR DESCRIPTION
**Pull request by andrewstech**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16812)

### CODE REVIEW

File: manifests/b/BritishBroadcastingCorporation/Iplayer/2.13.2/BritishBroadcastingCorporation.Iplayer.locale.en-GB.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BritishBroadcastingCorporation.Iplayer
```
Comment (ItzLevvie) [inline manifests/b/BritishBroadcastingCorporation/Iplayer/2.13.2/BritishBroadcastingCorporation.Iplayer.locale.en-GB.yaml:3]:
```suggestion
PackageIdentifier: BBC.iPlayer
```

File: manifests/b/BritishBroadcastingCorporation/Iplayer/2.13.2/BritishBroadcastingCorporation.Iplayer.locale.en-GB.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BritishBroadcastingCorporation.Iplayer
+PackageVersion: 2.13.2
+PackageLocale: en-GB
+Publisher: British Broadcasting Corporation
+PackageName: Iplayer
```
Comment (ItzLevvie) [inline manifests/b/BritishBroadcastingCorporation/Iplayer/2.13.2/BritishBroadcastingCorporation.Iplayer.locale.en-GB.yaml:7]:
```suggestion
PackageName: BBCiPlayerDownloads
```

File: manifests/b/BritishBroadcastingCorporation/Iplayer/2.13.2/BritishBroadcastingCorporation.Iplayer.locale.en-GB.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BritishBroadcastingCorporation.Iplayer
+PackageVersion: 2.13.2
+PackageLocale: en-GB
+Publisher: British Broadcasting Corporation
+PackageName: Iplayer
+License: Copyright British Broadcasting Corporation MMXX
```
Comment (ItzLevvie) [inline manifests/b/BritishBroadcastingCorporation/Iplayer/2.13.2/BritishBroadcastingCorporation.Iplayer.locale.en-GB.yaml:8]:
```suggestion
License: Copyright (c) British Broadcasting Corporation
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my review.

**Comment by andrewstech** [comment]
> Please see my review.

Done :)

**Comment by ItzLevvie** [comment]
You'll need to update the folders as well.

**Comment by andrewstech** [comment]
> You'll need to update the folders and file names to match `BBC.iPlayer` as well.
> 
> It's easy to update it via the GitHub Desktop client on Windows.

Ah ok

**Comment by ItzLevvie** [comment]
You forgot about the file names :)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**